src/_data/LiveData.js

```diff
@@ -43,6 +43,13 @@ export const changeLanguage = langCode => {
     api.getTradingTimes(new Date());
 };
 
+export const getTickHistory = async (symbol) => {
+    try {
+        await api.getTickHistory(symbol, { end: 'latest', count: 10, subscribe: 1, adjust_start_time: 1 });
+    } catch (err) {
+        await api.getTickHistory(symbol, { end: 'latest', count: 10 });
```

Comment:
I'm much more in favor of using the subscribe separately.
Then it will be:

```
getTickHistory()
try {
  subscribeToTicks();
catcth() {
 // suppress error, do nothing
}